src/libcore/tests/test_xml.py

```diff
@@ -298,3 +298,166 @@ def test20_upgrade_tree(variant_scalar_rgb):
                                <float name="intIOR" value="1.33"/>
                            </bsdf>
                        </scene>""")
+
```

Comment:
We could move those tests in a different test file as this one is pretty large already.